dlk/python/dlk/templates/src/func/arch/aarch64_neon/quantized_conv2d.cpp

```diff
@@ -21,10 +21,10 @@ limitations under the License.
 #include <iterator>
 
 #include "global.h"
-#include "func/impl/apply_thresholds.h"
-#include "func/impl/quantized_conv2d_dim2col.h"
-#include "func/impl/quantized_conv2d_tiling.h"
-#include "func/impl/quantized_conv2d_kn2row.h"
+#include "func/apply_thresholds.h"
+#include "func/arch/quantized_conv2d_dim2col.h"
+#include "func/arch/quantized_conv2d_tiling.h"
+#include "func/arch/quantized_conv2d_kn2row.h"
```

Comment:
What rule are you using to decide should header become `func/xxx.h` or `func/arch/xxx.h` ? :thinking: 

Inputitively `arch/<arch name>` level should contain only implementations, while declarations should be somewhere else...

Reply:
Got explanations offline

Reply:
For `/include/func/`, it is a policy to put TensorFlow functions etc., and `/include/include/func/arch/` to put architecture dependent system functions.